test/workflows/in/.keboola/manifest.json

```diff
@@ -1,8 +1,8 @@
 {
   "version": 2,
   "project": {
-    "id": %%TEST_KBC_PROJECT_ID%%,
-    "apiHost": "%%TEST_KBC_STORAGE_API_HOST%%"
+    "id": "12345",
```

Comment:
Pre prikaz `workflows` sa nekontrolovala `version` v manifeste.
Teraz sa uz kontroluje a subor `manifest.json` musi byt validny.

`Workflows` nepotrebuje remote project, a preto sa tieto ENVs nenahradzovali.
Ale kedze sa manifest nenacitaval, tak to testu nevadilo.